docs/docs/Basics/Deployment.md

```diff
@@ -0,0 +1,111 @@
+---
+sidebar_position: 5
+---
+
+#### Prerequisites
+
+1. We do not recommend deploying a SQLite database on serverless environments since the data wouldn't be persisted. I provisioned a quick Postgresql database on [Railway](https://railway.app), but you can of course use any other database provider. Make sure the prisma schema is updated to use the correct database.
```

Comment:
this sentence is kinda dumb now that tools like Turso exists. Should probably rephrase that it's the on-disk file-solution you shouldn't use in serverless 🤷🏼‍♂️

Reply:
I copy pasted it from the ReadMe though, I guess we will have to remove from there as well. 
Will check the new state of the repo again and make the necessary changes